torch/csrc/autograd/variable.cpp

```diff
@@ -141,10 +84,6 @@ void Variable::remove_hook(unsigned pos) {
   (*list)[pos] = nullptr;
 }
 
-uint32_t Variable::current_version() const noexcept {
```

Comment:
What is the reason for changing this "public looking" interface to the more "private looking" interface `_version()`? Should the user not be able to access the current version?

Reply:
Solely for parity with what the Python API looks like. And `current_version` was never supposed to be a public API, it was accidentally made so on Variable.